linux/CMakeLists.txt

```diff
@@ -11,6 +11,11 @@ set(PROJECT_NAME "webview_cef")
 project(${PROJECT_NAME} LANGUAGES CXX C)
 set(CMAKE_CXX_STANDARD 17)
 
+# Needed for making it run on arm64 Linux
+if(CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL "aarch64")
+  set(PROJECT_ARCH "aarch64")
+endif()
+
```

Comment:
I don't think we can have a problem here because I don't see how we can run cmake for different architectures for the same Flutter application build. So we should be fine here.  The reason we need this is probably this block:  
https://github.com/chromiumembedded/cef/blob/323cbdc5ea35b1b3dd6588ce0b542ad3a96fc3cb/cmake/cef_variables.cmake.in#L28  

On Linux arm64, `CMAKE_HOST_SYSTEM_PROCESSOR` is set to `aarch64`. That's probably why you ask for that in the `download.cmake` (git blame says you added line 3 there). But CEF checks for `arm64` as you see in the linked block.   
So actually, CEF does set this variable and we could have the same discussion there.

Reply:
The three lines of code I modified in the git blame were to fix the issue of copying resources after compilation.

My concern is that may cause other issues if you modify CMAKE-HOS_SYSTEM-PCESSOR in the foremost cmake file.
You can see the way I solved this problem was by modifying the cmake in the recognition system of CEF, which only affects CEF itself. The file is in the pre built package cef_binary_114.2.9+g1a97a28+chromium-114.0.5735.91_linuxarm64\cmake\cef_variables.cmake  line 30

```
#Determine the project architecture.
if(NOT DEFINED PROJECT_ARCH)
  if(("${CMAKE_HOST_SYSTEM_PROCESSOR}" STREQUAL "arm64") OR
      ("${CMAKE_HOST_SYSTEM_PROCESSOR}" STREQUAL "aarch64") OR
     ("${CMAKE_CXX_COMPILER_ARCHITECTURE_ID}" STREQUAL "ARM64"))
    set(PROJECT_ARCH "arm64")
  elseif(CMAKE_SIZEOF_VOID_P MATCHES 8)
    set(PROJECT_ARCH "x86_64")
  else()
    set(PROJECT_ARCH "x86")
  endif()
endif()
```

Reply:
I resolve this since we have this discussion below.

Reply:
> The three lines of code I modified in the git blame were to fix the issue of copying resources after compilation.
> 
> My concern is that may cause other issues if you modify CMAKE-HOS_SYSTEM-PCESSOR in the foremost cmake file. You can see the way I solved this problem was by modifying the cmake in the recognition system of CEF, which only affects CEF itself. The file is in the pre built package cef_binary_114.2.9+g1a97a28+chromium-114.0.5735.91_linuxarm64\cmake\cef_variables.cmake line 30
> 
> ```
> #Determine the project architecture.
> if(NOT DEFINED PROJECT_ARCH)
>   if(("${CMAKE_HOST_SYSTEM_PROCESSOR}" STREQUAL "arm64") OR
>       ("${CMAKE_HOST_SYSTEM_PROCESSOR}" STREQUAL "aarch64") OR
>      ("${CMAKE_CXX_COMPILER_ARCHITECTURE_ID}" STREQUAL "ARM64"))
>     set(PROJECT_ARCH "arm64")
>   elseif(CMAKE_SIZEOF_VOID_P MATCHES 8)
>     set(PROJECT_ARCH "x86_64")
>   else()
>     set(PROJECT_ARCH "x86")
>   endif()
> endif()
> ```

I made a change to the `download.cmake` that modifies the cmake file of CEF. How do you like that?